department-of-veterans-affairs/caseflow
# Queue | Return colocated tasks from AppealsController.tasks

Resolves #6517 

### Description
When loading the case detail view as a colocated user, we hit `/appeals/${appealId}/tasks?role=Colocated`, which would attempt and fail to get legacy tasks from `LegacyWorkQueue`, because `ColocatedTask`s have no `from_vacols` method / are post-VACOLS.

This change returns `ColocatedTask`s for colocated users when visiting `/queue/appeals/${appealId}`. It also renames the Flow type `Task` to `LegacyTask`

### Acceptance Criteria 
- [ ] `http://localhost:3000/appeals/${appealId}/tasks?role=Colocated` returns tasks

### Testing Plan
1. Go to `/queue` as `BVALSPORER`, navigate to case detail view with dev tools open
2. Confirm response to `/appeals/${appealId}/tasks?role=Colocated` has task data (no UI implemented yet)
